src/utils/getImage.js

```diff
@@ -0,0 +1,29 @@
+const { default: axios } = require("axios");
```

Comment:
```suggestion
const axios = require("axios");
```
Hey @MizouziE,  I think it's the way you import axios that's way the tests are failing with `Cannot read properties of undefined (reading get)`.

Reply:
:man_facepalming: I can't believe I missed that!

Thank you man :pray: